Xamarin.Forms.DataGrid/DataGridColumn.cs

```diff
@@ -54,10 +54,10 @@ public string Title
 			set { SetValue(TitleProperty, value); }
 		}
 
-		public string PropertyName
+		public string PropertyPath
```

Comment:
Its a breaking change you shouldn't rename `PropertyName` to `PropertyPath`.If we rename this and once user update the nuget package, it will break UI codes at the end side.